website/api/helpers/get-extended-osquery-schema.js

```diff
@@ -73,6 +73,13 @@ module.exports = {
 
         expandedTableToPush.fleetRepoUrl = 'https://github.com/fleetdm/fleet/new/main/schema/tables/?filename='+encodeURIComponent('/tables/'+expandedTableToPush.name)+'.yml&value='+encodeURIComponent(sampleYamlSchemaForThisTable);
 
+        // Grab the last example in the array as is usually the most rich example.
+        // TODO: adjust the UI to show all examples.
+        let examples = expandedTableToPush.examples;
+        if (examples.length > 0) {
+          expandedTableToPush.examples = '```\n' + examples[examples.length- 1] + '\n```';
+        }
+
```

Comment:
I did this so the examples we pull from the osquery repo are rendered as code, attaching screenshots of before and after:

<img width="864" alt="image" src="https://user-images.githubusercontent.com/4419992/211089655-c8aa6525-576d-4500-b9e8-957461c9c84e.png">


<img width="876" alt="image" src="https://user-images.githubusercontent.com/4419992/211089631-1d095d35-aff2-4e9e-8389-be599627ca3e.png">

Reply:
@eashaw

Reply:
should this be on line 99?  Not sure fofhand, up to you @eashaw

Reply:
good point, probably in both places OR around line 60?

- this conditional branch covers the case when there aren't any overrides
- something on line 99 should cover the case when we have overrides for other fields, but not for the examples